sidekiq_adhoc_job.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.6.0'
 
   spec.add_development_dependency 'pry'
-  spec.add_development_dependency 'bundler', '~> 2.0.1'
+  spec.add_development_dependency 'bundler'
```

Comment:
To fix TravisCI failed build, for some reason TravisCI unable to find the older version of bundler gem.